### PR TITLE
Update benchmarking docs

### PR DESCRIPTION
**Pull request by dpino**
Ditto.

### CODE REVIEW

File: src/program/lwaftr/doc/README.benchmarking.md [hunk 1]
```diff
@@ -1,38 +1,37 @@
 # Benchmarking
 
-The instructions in [README.first.md](README.first.md) for running the lwaftr with a load
-generator are the instructions for the primary way to benchmark the lwaftr.
-In short:
+The instructions in [README.first.md](README.first.md) for running the lwaftr
```
Comment (wingo) [inline src/program/lwaftr/doc/README.benchmarking.md]: there is no README.first.md, maybe you mean README.running.md

File: src/program/lwaftr/doc/README.benchmarking.md [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
-First, start the lwAFTR:
+Please, change the PCI addresses to match the settings in your current system; 
+See [README.first.md](README.first.md).
+
+In one server, start the lwAFTR:
 
 ```
-$ sudo ./src/snabb-lwaftr run \
+$ sudo taskset -c 1 ./src/snabb-lwaftr run -v \
     --conf tests/apps/lwaftr/data/icmp_on_fail.conf \
```
Comment (wingo) [inline src/program/lwaftr/doc/README.benchmarking.md]: this path is incorrect

File: src/program/lwaftr/doc/README.benchmarking.md [hunk 3]
```diff
@@ -82,27 +83,72 @@ them to CPUs in the NUMA node 0.
...
 
+## Version 2.0
+
+Charts are not available at this moment.  Version 2.0 fixes packet loss
+for small binding tables, however there are still packet loss reported
+for big binding tables.  An excerpt of maximum peformance peak before 
+packets start to lose is pasted below.  See [loadtest.txt][benchmarks-v1.0/loadtest.txt]
+for a more detailed report.
+
+```
```
Comment (wingo) [inline src/program/lwaftr/doc/README.benchmarking.md]: We need a result with small binding tables too showing that we do in fact reach line rate.

File: src/program/lwaftr/doc/README.benchmarking.md [hunk 4]
```diff
@@ -1,38 +1,37 @@
...
-    --v4-pci 0000:01:00.1 --v6-pci 0000:02:00.1
+$ sudo taskset -c 1 ./src/snabb-lwaftr run -v \
+    --conf program/lwaftr/tests/data/icmp_on_fail.conf \
+    --v4-pci 0000:02:00.0 --v6-pci 0000:02:00.1
 ```
 
-The lwAFTR won't print out much to the console.  You can pass the `-v`
-flag to enable periodid printouts if you like.
+The `-v` flag enables periodic printouts reporting MPPS and Gbps statistics per
+NIC.
```
Comment (kbara) [inline src/program/lwaftr/doc/README.benchmarking.md]: Is this a thing that handles -v -v too, or not?

Comment (dpino) [inline src/program/lwaftr/doc/README.benchmarking.md]: It accepts an number to increase verbosity.

Comment (kbara) [inline src/program/lwaftr/doc/README.benchmarking.md]: That should be mentioned in this part of the docs then, with an example.

**Pull request merged**